test/BUILD.gn

```diff
@@ -1118,6 +1117,7 @@ test("brave_browser_tests") {
     deps += [
       "//brave/app/theme:brave_theme_resources_grit",
       "//brave/app/theme:brave_unscaled_resources_grit",
+      "//brave/browser/request_otr:browser_tests",
```

Comment:
add this to the list in android_browser_tests.gni

Reply:
done in https://github.com/brave/brave-core/commit/c83ae23838b8132c40d9ce007c338abf52804ae9